app/views/home/_navbar.html.erb

```diff
@@ -1,18 +1,26 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
+<nav class="navbar navbar-expand-lg navbar-light" style="background-color: #e3f2fd;">
```

Comment:
In-line styles are fine in React, but outside, you want to put this in the CSS file for performance and maintainability reasons.